acceptance_tests/features/steps/rh_ui_cookies.py

```diff
@@ -0,0 +1,118 @@
+from behave import step
+
+import json
+
+from acceptance_tests.utilities.test_case_helper import test_helper
+from config import Config
+
+RETURN = "\ue006"
+
+
+@step("the cookies page is displayed")
+def display_uac_cookies_page(context):
+    context.browser.visit(f"{Config.RH_UI_URL}en/cookies")
+
+
+# Cookies banner checks and interactions
+
+
+@step("the cookies banner is displayed")
+def display_cookies_banner(context):
+    context.browser.cookies.delete_all()  # clear cookies to ensure that the banner is displayed
```

Comment:
Is this line needed? It's deleting the cookies then finding the banner but there's nothing to actually reload the page.